src/Token/AccessToken.php

```diff
@@ -41,7 +46,11 @@ class AccessToken implements JsonSerializable
     protected $resourceOwnerId;
 
     /**
-     * @param array $options
+     * Constructs an access token.
+     *
+     * @param array $options An array of options returned by the service provider
+     *     in the access token request. The `access_token` option is required.
```

Comment:
I tend to agree, but I go back-and-forth on my own arguments on this.

As it is, it took me a while to write up these descriptions, since I had to sift through the code to make sense of it. In fact, looking at the code, you can see that you're able to pass in `grantFactory`, `requestFactory`, `httpClient`, and `randomFactory` as options, too, and they are properly set, but then they are overridden if they aren't provided as collaborators, so I listed them in the comment to make it clear to implementors where they should go.